wdl-lint/src/rules/matching_parameter_meta.rs

```diff
@@ -102,6 +102,18 @@ impl Rule for MatchingParameterMetaRule {
             SyntaxKind::ParameterMetadataSectionNode,
         ])
     }
+
+    fn related_rules(&self) -> &[&'static str] {
+        &[
+            "DescriptionMissing",
+            "MissingMetas",
```

Comment:
Remove `MissingMetas` because if this rule is firing, the user has already written meta sections